app/edit/layout.tsx

```diff
@@ -1,13 +1,7 @@
-export default function RunLayout({
-                                      children,
-                                  }: {
-    children: React.ReactNode;
-}) {
+export default function EditLayout({children,}: {children: React.ReactNode}) {
```

Comment:
nit: I _think_ this comma can be removed

```suggestion
export default function EditLayout({children}: {children: React.ReactNode}) {
```